is-logged-in.bs

```diff
@@ -18,18 +18,184 @@ Level: None
 
 <em>This section is non-normative.</em>
 
-Blah blah blah.
+Currently, user agents have no way of knowing if the user is logged in
+to a particular website. Neither the existence of cookies nor
+frequent/recent user interaction can provide that signal, since most users
+have cookies for and interact with plenty of websites they are not
+logged in to. Additionally, cookies and other kinds of storage may carry login state, but there is no way to tell general storage and authentication tokens apart.
+
+At the same time, some user agents may wish to offer means to clear long-term storage where not the user is logged in, considering it a privacy issue that websites can store data virtually forever on the device. A logged-in signal could also enable a user agent to offer powerful browser features, or to create a seamless user experience by relaxing particular restrictions on websites where the user is logged in.
+
 </section>
 
 <h2 id="infra">Infrastructure</h2>
 
 This specification depends on the Infra standard. [[!INFRA]]
 
-<h2 id="acknowledgements" class="no-num">Acknowledgements</h2>
+<h2 id="the-isloggedin-api">The IsLoggedIn API</h2>
+
+The IsLoggedIn API creates a clear signal to the user agent that the user is logged into a website. It does <em>not</em> create a mechanism for the web developer to manage a user's identity or log the user in or out.
+
+This specification defines:
+
+  * A method for a website to signal to the browser that the user is logged in ({{Navigator/setLoggedIn()}})
+  * A method to signal that the user is logged out ({{Navigator/setLoggedOut()}})
+  * A method for third-party iframes to check whether the user is a logged-in customer ({{Navigator/setLoggedOut()}})
+
+ISSUE: Update "website" with a more precise term
+
+<h3 id="setLoggedIn">Logged-in signal: <dfn export method for=Navigator><code>setLoggedIn()</code></dfn></h3>
+
+ISSUE: Flesh out section with explanatory text, an example, more formal/algorithmic language.
+
+```
+Navigator.setLoggedIn(
+    username: non-whitespace string of limited length,
+    credentialTokenType: “httpStateToken” OR “legacyAuthCookie”,
```

Comment:
We've received https://github.com/privacycg/is-logged-in/issues/8 which asks that we don't refer to cookies as legacy (yet). I can only assume that what we call them will be what we get for the future too.